dev/ProgressBar/ProgressBar_themeresources.xaml

```diff
@@ -32,6 +35,7 @@
             <Color x:Key="SystemControlErrorBackgroundColor">#29C50500</Color>
             <!-- Hex value for SystemErrorText at 0.16 opacity -->
             <contract7NotPresent:Color x:Key="SystemErrorTextColor">#C50500</contract7NotPresent:Color>
+            <contract7NotPresent:SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Color="#C50500"/>
```

Comment:
>C50500 [](start = 103, length = 6)

These brushes should have their color set to the SystemErrorTextColor, not its value.

Reply:
<SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Color="{StaticResource SystemErrorTextColor}" />

---
In reply to: [491099136](https://github.com/microsoft/microsoft-ui-xaml/pull/3201#discussion_r491099136) [](ancestors = 491099136)

Reply:
Fixed.

Reply:
The high contrast version in system xaml actually points to SystemColorWindowTextColor instead of SystemErrorTextColor.

Reply:
When trying to reference that resource, I get the same visual state bugs we got earlier in this PR. Any ideas what could lead to that resource not being found? A few control in OS XAML reference that brush, so this shouldn't be an issue right?

Reply:
Sorry, its been a while, can you remind me what the issue was?

Reply:
@chingucoding ping

Reply:
Oh I am very sorry, thought that I already replied. The issue I am encountering is that the visual states of the ProgressBar seem to be missing resulting in crashes when visiting the page. As mentioned in my comment earlier, it seems that it fails to find the themeresource "SystemColorWindowTextColor". Should we add the resource to the ProgressBar_themeresources.xaml file to mitigate this?